### PR TITLE
feat: Sell Mode Button

### PR DESCRIPTION
**Pull request by cbachmeier**
- adds button to profile page to go into Sell Mode, this allows cards to be selected and added to the listing tag
  - Select all button only visible in Sell Mode
  - Toggling Sell Mode reset sell state
- remove sort and search which will not be included in phase1 launch

### CODE REVIEW

File: src/nft/components/profile/view/WalletAssetDisplay.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+import { VerifiedIcon } from 'nft/components/icons'
+import { subheadSmall } from 'nft/css/common.css'
+import { useBag, useIsMobile, useSellAsset } from 'nft/hooks'
+import { WalletAsset } from 'nft/types'
+import { useMemo, useReducer } from 'react'
+import { Link } from 'react-router-dom'
+
+import * as styles from './ProfilePage.css'
+
+const formatEth = (price: number) => {
```
Comment (JackShort) [inline src/nft/components/profile/view/WalletAssetDisplay.tsx:12]: I think you previously made a util function for this

File: src/nft/components/profile/view/WalletAssetDisplay.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        (x) => x.tokenId === asset.tokenId && x.asset_contract.address === asset.asset_contract.address
+      ) &&
+      !isMobile
+    )
+      toggleCart()
+  }
+
+  return (
+    <Link
+      to={`/nfts/asset/${asset.asset_contract.address}/${asset.tokenId}?origin=profile`}
```
Comment (JackShort) [inline src/nft/components/profile/view/WalletAssetDisplay.tsx:55]: `getAssetHref` in `asset.ts` util can generate this for you

File: src/nft/components/profile/view/WalletAssetDisplay.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        onMouseEnter={toggleBoxHovered}
+        onMouseLeave={toggleBoxHovered}
+      >
+        <Box
+          as="img"
+          alt={asset.name}
+          width="full"
+          borderTopLeftRadius="20"
+          borderTopRightRadius="20"
+          src={asset.image_url || '/nft/svgs/image-placeholder.svg'}
```
Comment (JackShort) [inline src/nft/components/profile/view/WalletAssetDisplay.tsx:70]: `??` better for nullish checks

**Review by JackShort — APPROVED** [review]
lgtm - non blocking changes

File: src/nft/components/profile/view/WalletAssetDisplay.tsx (hunk 2 above)
Comment (cbachmeier) [reply src/nft/components/profile/view/WalletAssetDisplay.tsx:55]: I've modified the function to take a WalletAsset as input as well

**Review by JackShort — APPROVED** [review]
nice

**Pull request merged**